pkg/services/sqlstore/dashboard_test.go

```diff
@@ -481,19 +482,21 @@ func insertTestDashboardForPlugin(title string, orgId int64, folderId int64, isF
 	return cmd.Result
 }
 
-func createUser(name string, role string, isAdmin bool) models.User {
+func createUser(t *testing.T, name string, role string, isAdmin bool) models.User {
+	t.Helper()
+
 	setting.AutoAssignOrg = true
 	setting.AutoAssignOrgId = 1
 	setting.AutoAssignOrgRole = role
 
 	currentUserCmd := models.CreateUserCommand{Login: name, Email: name + "@test.com", Name: "a " + name, IsAdmin: isAdmin}
 	err := CreateUser(context.Background(), &currentUserCmd)
-	So(err, ShouldBeNil)
+	require.NoError(t, err)
 
 	q1 := models.GetUserOrgListQuery{UserId: currentUserCmd.Result.Id}
 	err = GetUserOrgList(&q1)
-	So(err, ShouldBeNil)
-	So(q1.Result[0].Role, ShouldEqual, role)
+	require.NoError(t, err)
+	require.Equal(t, q1.Result[0].Role, models.RoleType(role))
```

Comment:
The expectation comes before the actual value in `require.Equal`:
```suggestion
	require.Equal(t, models.RoleType(role), q1.Result[0].Role)
```

Reply:
Ah, yes. Fixed. Thanks!